bip-0001.mediawiki

```diff
@@ -149,21 +153,23 @@ A BIP editor must subscribe to the Bitcoin development mailing list. All BIP-rel
 
 For each new BIP that comes in an editor does the following:
 
-* Read the BIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to be accepted.
+* Read the BIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to be accepted and must be relevent to Bitcoin.
 * The title should accurately describe the content.
 * Edit the BIP for language (spelling, grammar, sentence structure, etc.), markup (for reST BIPs), code style (examples should match BIP 8 & 7).
 
 If the BIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
 
-Once the BIP is ready for the repository, the BIP editor will:
+Once the BIP is ready for the repository it should be submitted as a "pull request" to the [https://github.com/bitcoin/bips bitcoin/bips] repository on GitHub where it may get further feedback.
+
+The BIP editor will:
 
-* Assign a BIP number (almost always just the next available number, but sometimes it's a special/joke number, like 666 or 3141).
+* Assign a BIP number (almost always just the next available number, but sometimes it's a special/joke number, like 666 or 3141) in the pull request comments.
 
-* Add the BIP to the [https://github.com/bitcoin/bips bitcoin/bips] repository on GitHub.
+* Merge the pull request when the author is ready (allowing some time for further peer review).
```

Comment:
Review was supposed to occur on the Bitcoin-Development ML before being submitted as a PR. Once it's a PR, it should be ready for merging immediately (pending author ACK, if submitted by non-author).

If we want to encourage PR discussion, adding a flag like [WIP] makes sense.